projects/lib/src/oauth-service.ts

```diff
@@ -2263,8 +2263,8 @@ export class OAuthService extends AuthConfig implements OnDestroy {
     const clockSkewInMSec = this.getClockSkewInMsec(); // (this.getClockSkewInMsec() || 600) * 1000;
 
     if (
-      issuedAtMSec - clockSkewInMSec >= now ||
-      expiresAtMSec + clockSkewInMSec <= now
+      issuedAtMSec >= now - clockSkewInMSec ||
```

Comment:
I think the sign of the clockSkew is wrong when comparing with the issuedAt.
In the case when the Identity Provider server clock is the same as the the browser clock, the issuedAt of a refreshed access_token/id_token will be very close to the server time.
Reducing the browser time with the clock skew, it will lead to the "token has expired" error each time, no?

```suggestion
      issuedAtMSec >= now + clockSkewInMSec ||
```

Reply:
I think you are right. The original code is correct here.

Reply:
The original code was correct only for issuedAt attribute.
But, the updates you suggest on the expiresAt seem correct to me